src/calculateRentalCost.js

```diff
@@ -1,10 +1,18 @@
-/**
+/*
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PriseDay = 40;
+
+  if (days < 3) {
+    return days * PriseDay;
+  } else if (days >= 3 && days < 7) {
```

Comment:
The check `days >= 3` is redundant because if the code execution reached this point, `days < 3` has already been evaluated as false in the previous `if` statement.